lib/ansible/modules/cloud/helm.py

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/python
+# (c) 2016, Flavio Percoco <flavio@redhat.com>
+#
+# This module is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this software.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: helm
+short_description: Manages Kubernetes packages with the Helm package manager
+version_added: "2.4"
+author: "Flavio Percoco (flaper87)"
+description:
+   - Install, upgrade, delete and list packages with the Helm package manage
+requirements:
+  - "pyhelm"
+  - "grpcio"
+options:
+  host:
+    description:
+      - Tiller's server host
+    required: false
+    default: "localhost"
+  port:
+    description:
+      - Tiller's server port
+    required: false
+    default: 44134
+  namespace:
+    description:
+      - Kubernete's namespace where the chart should be installed
+    required: false
+    default: "default"
+  name:
+    description:
+      - Release name to manage
+    required: false
+    default: null
+  state:
+    description:
+      - Whether to install (C(present) or C(installed)), or remove (C(absent) or C(removed)) a package.
+    required: false
+    choices: ['absent', 'present', 'installed', 'removed']
+    default: "installed"
+  chart:
+    description: |
+      A map describing the chart to install. For example:
+      chart:
+        name: memcached
+        version: 0.4.0
+        source:
+          type: repo
+          location: https://kubernetes-charts.storage.googleapis.com
+    required: false
+    default: {}
+  values:
+    description:
+      - A map of value options for the chart.
+    required: false
+    default: {}
+  disable_hooks:
+    description:
+      - Whether to disable hooks during the uninstall process
+    required: false
+    default: false
+  purge:
```

Comment:
Since this is more of a special kind of "absent", maybe it would be better to have two "absent" states - absent and purged - since it seems odd to be allowed to specify "present + purged=true".